addons/l10n_in/data/template/account.account-in.csv

```diff
@@ -89,3 +89,7 @@
 "p300010","Salary Exp Payable","300010","liability_current","","True"
 "p300011","Leave Travel Allowance Expense","300011","liability_current","","False"
 "p300012","Professional Tax Payable","300012","liability_current","","False"
+"p213321","IGST Expense","213321","expense","l10n_in.igst_tag_account","False"
```

Comment:
Find that weird that we don't have the codes in order, i know that there is other account that are not in order but still :grimacing: